### ISSUE TITLE
Apply design to the "forgot password" page

### ISSUE THREAD

**Issue by Elian8or** [lukovdm/bf2-www#52]
What are the fill-in options, only their emailadress?

**Comment by lukovdm** [comment]
Yes correct, and then a new page where they can change there password, but that can be the same as #51
